src/app/SBE/detail/company/index.js

```diff
@@ -101,22 +102,26 @@ const Rep = ({ sbeCode, targetCode }) => {
   )
 
   const digitalJobsButton =
-    validation?.value === 'Ready' || validation?.value === 'Validated' || digitalJobsValidation ? (
+    validation?.value === 'Ready' ||
+    validation?.value === 'Validated' ||
+    (digitalJobsAgreement === 'true' && digitalJobsValidation) ? (
```

Comment:
You can do only digitalJobsAgreement or !!digitalJobsAgreement instead of digitalJobsAgreement === 'true'